guide/designing-products/user-research-resources.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: guide
+title: User research resources
```

Comment:
Change to just "Resources" to match URL structure. Not a biggie though.

Reply:
Wouldn't that be weird for people who randomly end up on this page, not knowing the context? Especially on mobile, you wouldn't see that the page is nested under a "User research" page.

Reply:
I like the idea of having this page just called 'Resources' and internally is split into:

1. Research
2. Tools (things designers may find useful) 
3. Other things like the Bitcoin Icon set / wallet UI Kit. 

Styling could be simple just with the Heading, such as 'Research' with an unordered list underneath linking externally to the resource.

Reply:
I'd like to keep this PR focused on just putting together a decent page about user research. 

For future PRs, I'm personally more drawn to keeping this page focused on user research and adding good summaries of each project. It's pretty intimidating to see a 20-page academic paper so I think a lot of research is not being used. I think it's worth seeing if we cannot pull out the main findings and present them as TLDRs so they can be referenced by designers and maybe also serve as discussion material (on Slack and Twitter).

@Bosch-0 what are some examples of Tools? For your third point, the icon set is already linked to from the [Projects](https://bitcoin.design/projects/) page. There's also the [old concept of the design resource page](https://github.com/BitcoinDesign/Guide/issues/35). Those are good things to build out further, we just need to have clear places for each type of resource.

Reply:
Yeah fair point, I'm in favour of the current format and then looking at ways to display the key findings in each piece of research.